js/web/lib/wasm/jsep/webgpu/ops/matmul.ts

```diff
@@ -91,23 +87,32 @@ export const createNaiveMatmulProgramInfo = (
     ];
     appendActivationUniforms(activationAttributes, uniforms);
 
-    const getIndices = (variable: IndicesHelper, broadCastDims: number[]) => {
-      const rank = variable.rank;
-      const name = variable.name;
-      if (rank === 2) {
-        return `var ${name}_indices = ${variable.type.indices}(0u, 0u);`;
+    // Helper that convert output batch to input batch indices using only the rank and the shape information in uniform
+    const convertOutputBatchIndicesToInput = (inputVariable: IndicesHelper, targetIndicesName: string) => {
```

Comment:
Is it possible to reuse below one to reduce some duplicated code?https://github.com/microsoft/onnxruntime/blob/6d225e01319faa99cb882cb3623d55821cc9d487/js/web/lib/wasm/jsep/webgpu/ops/3rd-party/matmul_packed_webgpu.ts#L382-L410

Reply:
Done, make the helper general in `matmul.ts` and reusing it in `3rd-party/matmul-packed-webgpu.ts`.